docs/2.7.0/docs/daml-finance/tutorials/getting-started/holding.rst

```diff
@@ -0,0 +1,194 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Holding
+#######
+
+This tutorial introduces the core asset model of the library through a simple example. The purpose
+is to illustrate the concepts of :ref:`account <account>`, :ref:`instrument <instrument>`, and
+:ref:`holding <holding>`, as well as showing how to work with Daml interfaces.
+
+We are going to:
+
+#. create accounts for Alice and Bob at the Bank
+#. issue a cash instrument
+#. credit a cash holding to Alice’s account
+
+We expect the reader to be familiar with the basic building blocks of Daml. If that is not the case,
+a suitable introduction can be found `here <https://www.digitalasset.com/developers/learn>`_.
+In particular, `An Introduction to Daml <https://docs.daml.com/daml/intro/0_Intro.html>`_
+would be a good starting point.
+
+This example refers to the :doc:`Transfer <transfer>` tutorial script in the sense that it explains
+the accounts and holdings required for that tutorial.
+
+Explanation of Holdings
+***********************
+
+As described in the :ref:`Asset model <holding-asset-model>`, a holding contract represents the
+ownership of a certain amount of an :ref:`instrument <instrument>` by an owner at a custodian.
+
+A holding implementation can have specific properties such as being :ref:`fungible <fungibility>` or
```

Comment:
or non-transferable

Reply:
you mean "transferable or non-transferable"?